Neos.Flow/Classes/Mvc/Routing/Route.php

```diff
@@ -72,6 +73,20 @@ class Route
      */
     protected $appendExceedingArguments = false;
 
+    /**
+     * Specifies the tags to be associated with cache items generated by this route
+     *
+     * @var RouteTags|null
+     */
+    protected $cacheTags = null;
```

Comment:
Is there a specific reason for not adding type hints like
```suggestion
    protected ?RouteTags $cacheTags = null;
```
?
To stay in sync with the rest of the file?

Reply:
Yes it would look foreign in there. That is also why the new Route Lifetime DTO looks more like the RouteTags